website/docs/r/virtual_machine.html.markdown

```diff
@@ -423,6 +424,7 @@ resource "azurerm_virtual_machine" "test" {
 * `disk_size_gb` - (Required) Specifies the size of the data disk in gigabytes.
 * `caching` - (Optional) Specifies the caching requirements.
 * `lun` - (Required) Specifies the logical unit number of the data disk.
+* `write_accelerator_enabled` - (Optional) Specifies if Write Accelerator is enabled on the disk. This can only be enabled on `Premium_LRS` managed disks with no caching and [M-Series VMs](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/how-to-enable-write-accelerator). Defaults to false.
```

Comment:
_minor_ can we quote `false`?

Reply:
Hi 
is this the write way to use?
write_accelerator_enabled = true
getting the following error
invalid or unknown key: write_accelerator_enabled

Please advise